tests/contracts/crowdfunding_proc.scilla

```diff
@@ -0,0 +1,194 @@
+(***************************************************)
+(*                 Scilla version                  *)
+(***************************************************)
+
+scilla_version 0
+
+(***************************************************)
+(*               Associated library                *)
+(***************************************************)
+library Crowdfunding
+
+let andb = 
```

Comment:
While we are at it, could we reuse `andb`, `orb` and `negb` from the standard library (`BoolUtils`)?

Reply:
I was about to. There are also a couple of other things I would like to change for the purposes of illustrating library code in the documentation.

Reply:
+1! Please feel free :)

Reply:
Fixed.

I tried to add IntUtils as well (because we use `lt` and `negb` to test for greater-than in the contract), but that ended up costing an extra 400 gas for each transition due to all the unused functions that need to be instantiated to closures.